package.json

```diff
@@ -15,12 +15,13 @@
   "scripts": {
     "build": "babel src -d dist",
     "prepack": "npm run build",
-    "test": "mocha"
+    "test": "mocha --timeout 10000"
```

Comment:
some specs run for longer than the default timeout so this gives us room for error

Reply:
This is redundant with the `.mocharc.yml` setting, I would remove from here.